### PR TITLE
aww yeah baby

### PR DESCRIPTION
**Pull request by andrewelizondo**
as of automate 0.6.7 or later we don't need chef gate to proxy audits through the chef server

### CODE REVIEW

**Comment by andrewelizondo** [comment]
@cheeseplus 👀

File: generator_files/cookbooks/chef_server/recipes/default.rb [hunk 1]
```diff
@@ -41,7 +41,12 @@
 
 chef_ingredient 'chef-server' do
   action :reconfigure
-  config "api_fqdn 'chef.#{node['demo']['domain']}'"
+  config <<-EOH
+  api_fqdn 'chef.#{node['demo']['domain']}'
+  data_collector['root_url'] = 'https://#{node['demo']['domain_prefix']}automate.#{node['demo']['domain']}/data-collector/v0/'
+  data_collector['token'] = '93a49a4f2482c64126f7b6015e6b0f30284287ee4054ff8807fb63d9cbd1c506'
```
Comment (scarolan) [inline generator_files/cookbooks/chef_server/recipes/default.rb]: Should this token become a node attribute?

**Review by scarolan — APPROVED** [review]
👍

File: generator_files/cookbooks/chef_server/recipes/default.rb (hunk 1 above)
Comment (andrewelizondo) [reply generator_files/cookbooks/chef_server/recipes/default.rb]: its the default token that we wont really ever change, we could make it an attribute though to remove repetition

**Comment by andrewelizondo** [comment]
@scarolan better?

**Pull request merged**